src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/IISUrlRewriteRule.cs

```diff
@@ -13,16 +13,19 @@ public class IISUrlRewriteRule : IRule
         public UrlMatch InitialMatch { get; }
         public IList<Condition> Conditions { get; }
         public UrlAction Action { get; }
+        public bool Global { get; }
 
         public IISUrlRewriteRule(string name,
             UrlMatch initialMatch,
             IList<Condition> conditions,
-            UrlAction action)
+            UrlAction action,
+            bool? global = null)
```

Comment:
Instead of `bool? global = null`, why not `bool global = false`? The  `.ctor` is falling back to `false` anyways. Also, why not `IsGlobal` instead?